SingularityService/src/test/java/com/hubspot/singularity/mesos/SingularityMesosTaskBuilderTest.java

```diff
@@ -138,6 +141,40 @@ public void testJobUserPassedAsEnvironmentVariable() {
   }
 
   @Test
+  public void testEnvironmentVariableOverrides() {
+    Map<String, String> overrideVariables = new HashMap<>();
+    overrideVariables.put("HS_ACCESS_GROUP", "test");
```

Comment:
generally try to keep HubSpot-specific things out of Singularity, even for tests ;)

Reply:
👍 Good catch - didn't think of that